integration-tests/emulated/helpers/src/lib.rs

```diff
@@ -404,3 +406,69 @@ macro_rules! test_chain_can_claim_assets {
 		}
 	};
 }
+
+/// note: $asset needs to be prefunded outside this function
+#[macro_export]
+macro_rules! create_pool_with_native_on {
```

Comment:
do we not have this defined somewhere in the SDK so we can reuse instead of duplicating code here?

Reply:
It is in the BridgeHub Westend integration tests crate: https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-westend/src/tests/mod.rs#L126

So I thought it would be weird to use it here. I think it should be moved to a common crate in the `polkadot-sdk` though, will make a PR and add a comment here to replace it eventually.

Reply:
https://github.com/paritytech/polkadot-sdk/pull/8289